code/modules/mob/living/simple_animal/hostile/necro.dm

```diff
@@ -333,7 +335,6 @@
 /mob/living/simple_animal/hostile/necro/zombie/proc/eat(var/mob/living/carbon/human/target)
 	//Deal a random amount of brute damage to the corpse in question, heal the zombie by the damage dealt halved
 	visible_message("<span class='warning'>\The [src] takes a bite out of \the [target].</span>")
-	busy = TRUE
```

Comment:
because it's a leftover remnant from when there was a do_after here, now it does nothing but can override the busy check that's used in door opening